HopsanGUI/GUIObjects/GUIContainerObject.cpp

```diff
@@ -3499,3 +3491,22 @@ LogDataHandler2 *ContainerObject::getLogDataHandler()
     return mpModelWidget->getLogDataHandler();
     //return mpLogDataHandler;
 }
+
+QStringList ContainerObject::getRequiredComponentLibraries() const
+{
+    QStringList requiredLibraryPaths;
```

Comment:
Why is it called "Paths"? Shouldn't it be "requiredLibraryIds"?

Reply:
Yes this is leftover from before introducing id. I forgot to change name. I will fix before merging